components/compositing/Cargo.toml

```diff
@@ -53,5 +53,5 @@ git = "https://github.com/servo/rust-core-text"
 git = "https://github.com/servo/gleam"
 
 [dependencies]
-url = "*"
-time = "*"
\ No newline at end of file
+url = "0.2.16"
```

Comment:
I’m not sure you think this means what I think it means.

[Cargo docs](http://doc.crates.io/crates-io.html#using-crates.io-based-crates) say this is equivalent to `>= 0.2.16 < 0.3.0`, not `=0.2.16`.

That said, I don’t think we should use `=0.2.16`, that’s the job of `Cargo.lock`. See https://github.com/servo/servo/issues/4780#issuecomment-72321948